ocd-1903.sql

```diff
@@ -0,0 +1,38 @@
+UPDATE openchpl.global_user_permission_map as gupm
+SET user_permission_id_user_permission = 2
+WHERE user_id = 22;
+
+DELETE FROM openchpl.global_user_permission_map as gupm
+WHERE user_permission_id_user_permission = 3;
```

Comment:
Instead of deleting them, shouldn't they be updated to be the permissions 2 or 4, from 3/5?

Reply:
Agree with this. You should make an ACB_STAFF or ATL_STAFF user of your own, run this script, and make sure you can still log in with that user account. I don't think you'd be able to in this case because this would have deleted their permissions.